### ISSUE TITLE
arm64 docker container

### ISSUE THREAD

**Issue by alexolivier** [swagger-api/swagger-ui#8156]
### Content & configuration

Currently, the distributed containers are amd64 only and fail on:

```
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: IPv6 listen already enabled
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/40-swagger-ui.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2022/08/15 12:52:47 [emerg] 136#136: io_setup() failed (38: Function not implemented)
```

### Is your feature request related to a problem?
Publish an arm64 image.

**Comment by lucj** [comment]
Any update on that one ? arm64 image would be so nice when deploying swagger-ui in a local (Mac M1) Kubernetes cluster.

**Comment by tobiashochguertel** [comment]
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

Can we get a Arm64 docker image?

**Comment by zhitomir-oreshenski-mm** [comment]
Hi guys,

Does anyone knows if there is any progress or alternatives?

**Comment by gtowne** [comment]
We are also running into this issue. Published ARM images would be very nice.

**Comment by viktorsteinwand** [comment]
As I can see the base image https://hub.docker.com/_/nginx/tags?page=1&name=1.23.4-alpine is built for 3 different cpu architectures. So maybe just enabling image build for all archs will solve this issue?

**Comment by qeepcologne** [comment]
same for us, last image not available on arm64 and blocking migration to get rid of the expensive amd64 architecture.

**Comment by viktorsteinwand** [comment]
However there is the following workaround in place - Clone this repository on M1/M2 Mac and just build it by your own by choosing the same image name and tag:

```sh
git clone git@github.com:swagger-api/swagger-ui.git
cd swagger-ui
docker build --tag swaggerapi/swagger-ui:latest .
```

**Comment by lucj** [comment]
In case this can help, I've just cloned the sources of v4.19.0 and built for amd64 and arm64.
The image is lucj/swagger-ui:v4.19.0
I'll make sure to automate the process and have a new image created after each new official swagger-ui release.
That's a pity such a little hack has not been done before on the official repo with a GitHub Action for instance :(
Example of .github action I use for a little project (https://github.com/lucj/argocd-helmfile-plugin/blob/main/.github/workflows/build-image.yml)

**Comment by joshuanianji** [comment]
The [docker-build-push.yml](https://github.com/swagger-api/swagger-ui/blob/master/.github/workflows/docker-build-push.yml) workflow builds and pushes the image for arm64 already, and from [v5 and up](https://hub.docker.com/r/swaggerapi/swagger-ui/tags?page=1&name=v5) arm64 is supported *for most versions*.

It's pretty trippy, but the latest version (as of right now) is only built for amd64. I think this is because the amd64 image is built on Jenkins, and the other platforms are built on GHA later (via workflow_dispatch or repository_dispatch) - see comments on https://github.com/swagger-api/swagger-ui/issues/5815

Basically, don't use the latest tag! As of right now, v5.1.3 works for me.

```docker
docker pull swaggerapi/swagger-ui:v5.1.3
```

**Comment by char0n** [comment]
Hi everybody,

> The [docker-build-push.yml](https://github.com/swagger-api/swagger-ui/blob/master/.github/workflows/docker-build-push.yml) workflow builds and pushes the image for arm64 already, and from [v5 and up](https://hub.docker.com/r/swaggerapi/swagger-ui/tags?page=1&name=v5) arm64 is supported for most versions.

This is right. As of SwaggerUI@5, the [swagger-ui docker image](https://hub.docker.com/r/swaggerapi/swagger-ui) comes with full multi platform images. All the platforms that `nginx:1.x.y-alpine` supports, swagger-ui image support as well.

> It's pretty trippy, but the latest version (as of right now) is only built for amd64. I think this is because the amd64 image is built on Jenkins, and the other platforms are built on GHA later (via workflow_dispatch or repository_dispatch) - see comments on https://github.com/swagger-api/swagger-ui/issues/5815

Yes, it works exactly like that. Until we migrate from Jenkins entirely, Jenkins build the `amb64` and after Jenkins CICD pipeline is done, GitHub Action is triggered to append all the other platforms. Currently triggering the GitHub Action is manual, until https://github.com/swagger-api/swagger-ui/issues/8919 is implemented.

> Basically, don't use the latest tag! As of right now, v5.1.3 works for me.

v5.1.3 has multi platform support as well.


---

Overall, the multi platform support is fully there for SwaggerUI@5. We just don't provide multi platform images for `unstable` tag.
